ci/ray_ci/docker_container.py

```diff
@@ -71,14 +72,24 @@ def run(self) -> None:
     def _should_upload(self) -> bool:
         return os.environ.get("BUILDKITE_PIPELINE_ID") == POSTMERGE_PIPELINE
 
+    def _get_image_version_tags(self) -> List[str]:
+        branch = os.environ.get("BUILDKITE_BRANCH")
+        sha_tag = os.environ["BUILDKITE_COMMIT"][:6]
+        if branch == "master":
+            return [sha_tag, "nightly"]
+
+        if branch and branch.startswith("releases/"):
+            release_name = branch[len("releases/") :]
+            date_tag = datetime.now().strftime("%Y-%m-%d")
+            return [f"{release_name}.{sha_tag}", f"{release_name}.{date_tag}"]
```

Comment:
why do we need the date tagging?

we already have too many tags, and it is hard to tell if all date tags of one image are the same image.

Reply:
let's remove it!